.github/workflows/ci.yml

```
@@ -762,14 +762,20 @@ jobs:
         run: >
           breeze release-management generate-issue-content-providers
           --only-available-in-dist --disable-progress
+      - name: "Generate source constraints from CI image"
+        shell: bash
+        run: >
+          breeze release-management generate-constraints
+          --airflow-constraints-mode constraints-source-providers
       - name: "Install and verify all provider packages and airflow via wheel files"
         run: >
           breeze release-management verify-provider-packages
           --use-packages-from-dist
           --package-format wheel
           --use-airflow-version wheel
           --airflow-constraints-reference default
-          --providers-constraints-reference constraints-main
+          --providers-constraints-location
+          /files/constraints-"${{env.PYTHON_MAJOR_MINOR_VERSION}}"/constraints-source-providers-${{env.PYTHON_MAJOR_MINOR_VERSION}}.txt
```

```suggestion
          /files/constraints-${{env.PYTHON_MAJOR_MINOR_VERSION}}/constraints-source-providers-${{env.PYTHON_MAJOR_MINOR_VERSION}}.txt
```

Not sure if we need the quotes there? 🤔

---

We don't - it will work fine because we have never spaces in PYTHON_MAJOR_MINOR_VERSION. I removed it .

I also noticed we have another variant of this installation (when we have only some providers selected - not all of them - there we only install providers but do not run full verification for them as we don't have all providers ready)

I added it there too.